service/history/workflowWatcher.go

```diff
@@ -0,0 +1,122 @@
+// Copyright (c) 2017 Uber Technologies, Inc.
```

Comment:
2019

Reply:
This is generated code so I think the script should be updated rather than the generated code.